### PR TITLE
- Task with String functions

### PR DESCRIPTION
**Pull request by nadiiah**

### CODE REVIEW

File: src/main/java/com/chapter3/StringUtils.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        }
+        return count;
+    }
+
+    private static String findLetterIndex(String enteredString) {
+        StringBuilder letterIndexes = new StringBuilder();
+
+        for (int i = 0; i < enteredString.length(); i++) {
+            if (enteredString.charAt(i) == 'a') {
+                letterIndexes.append(i).append(" ");
```
Comment (iikarak) [inline src/main/java/com/chapter3/StringUtils.java]: add dependency to apache.commons  StringUtils.EMPTY

File: src/main/java/com/chapter3/StringUtils.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            }
+        }
+        return letterIndexes.toString();
+    }
+
+    private static String replaceLetters(String enteredString) {
+        return enteredString.replace('a', 'A');
+    }
+
+    private static String makeArrayFromString(String enteredString) {
```
Comment (iikarak) [inline src/main/java/com/chapter3/StringUtils.java]: add function wich will print each word in separate line and work with array[] as parameter

File: src/main/java/com/chapter3/StringUtils.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        return Arrays.toString(enteredString.split(" "));
+    }
+
+    private static String subtractText(String enteredString) {
+        int firstAIndex = enteredString.indexOf('a');
+        int lastAIndex = enteredString.lastIndexOf('a');
```
Comment (iikarak) [inline src/main/java/com/chapter3/StringUtils.java]: 'a' separete

File: src/main/java/com/chapter3/StringUtils.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+        for (int i = 0; i < enteredString.length(); i++) {
+            if (enteredString.charAt(i) == 'a') {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    private static String findLetterIndex(String enteredString) {
```
Comment (iikarak) [inline src/main/java/com/chapter3/StringUtils.java]: replace with using indexOf  with one and two parameters

File: src/main/java/com/chapter3/StringUtils.java [hunk 5]
```diff
@@ -0,0 +1,81 @@
+package com.chapter3;
+
+import java.util.Arrays;
+
+public class StringUtils {
```
Comment (iikarak) [inline src/main/java/com/chapter3/StringUtils.java]: гuse somewhere foreach

File: src/main/java/com/chapter3/ArrayUtils.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
+package com.chapter3;
+
+import java.util.Arrays;
+
+public class ArrayUtils {
+    public static void learnArrays() {
+        String[] names = new String[5];
+
+        for (int i = 0; i < 5; i++) names[i] = "";
```
Comment (iikarak) [inline src/main/java/com/chapter3/ArrayUtils.java]: bracates use in any cycle

File: src/main/java/com/chapter3/CycleUtils.java [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+        switch (choice) {
+            case 1:
+                System.out.println("One");
+                break;
+            case 2:
+                System.out.println("Two");
+                break;
+            case 3:
+                System.out.println("Three");
+                break;
```
Comment (iikarak) [inline src/main/java/com/chapter3/CycleUtils.java:18]: try to remove break

File: src/main/java/com/chapter3/StringUtils.java (hunk 3 above)
Comment (nadiiah) [reply src/main/java/com/chapter3/StringUtils.java]: Please remind what I should do here)

File: src/main/java/com/chapter3/StringUtil.java [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+        for (int i = 0; i < enteredString.length(); i++) {
+            if (enteredString.charAt(i) == 'a') {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    private static void findLetterIndex(String enteredString) {
+        int start = enteredString.indexOf('a');
```
Comment (iikarak) [inline src/main/java/com/chapter3/StringUtil.java]: or rename function to findALetterIndex, or path 'a' or any other letter as parameter

**Review by iikarak — APPROVED** [review]

**Pull request merged**